src/SFA.DAS.Payments.Audit.FundingSourceService/Infrastructure/Ioc/AuditFundingSourceServiceModule.cs

```diff
@@ -39,6 +43,16 @@ protected override void Load(ContainerBuilder builder)
             builder.RegisterType<FundingSourceEventModelHandler>()
                 .As<IHandleMessageBatches<FundingSourceEventModel>>()
                 .InstancePerLifetimeScope();
+            
+            builder.Register(c =>
```

Comment:
Seems like a lot of replicated code here, I think this registration could be added to the common audit autofac module in SFA.DAS.Payments.Audit.Application.Infrastructure.Ioc.AuditModule instead.